src/Core/src/Platform/Windows/MauiWebView.cs

```diff
@@ -157,5 +170,19 @@ void SetupPlatformEvents()
 				}
 			};
 		}
+
+		static bool IsUriWithLocalScheme(string? uri)
+		{
+			return uri?
+				.ToLowerInvariant()
+				.StartsWith(
+					LocalScheme.TrimEnd('/').ToLowerInvariant()) == true;
```

Comment:
Sorry I might not have been clear in an earlier comment. Does something like this work:

```suggestion
			return uri?.StartsWith(LocalScheme.TrimEnd('/'), StringComparison.Ordinal) == true;
```

And I suggest specifically using case-sensitive comparisons because URIs are fundamentally case-sensitive, so I think we should stick to that here.

Reply:
Agreed, and sorry I didn't know that `StartsWith()` has a string comparison parameter. I'll change that call.

Reply:
I pushed a changeset with the requested changes.